contractsdk/cpp/example/move.cc

```diff
@@ -0,0 +1,134 @@
+#include <cstdio>
+#include <cstdlib>
+#include <climits>
+#include "xchain/xchain.h"
+
+struct Move : public xchain::Contract {};
+
+const std::string BALANCEPRE = "balanceOf_";
+
+enum ret_t {
+    RET_SUCCESS = 0,  
+    RET_ERROR_INVALID_NUM,
+    RET_ERROR_OVERFLOW
+};
+
+ret_t string2num(const std::string& from, int64_t *to) {
+    long long temp;
+    temp = std::stoll(from.c_str(), NULL, 10);
```

Comment:
if non-digit char included in "from" like "123abc", 123 will be parsed to number. Better to pass a char *p to a pointer to the first non-digit char.

Reply:
Yes, you're right. Got it. The pointer will go to the next position of the numerical number. I'll fix it.